device-connectors/src/testflinger_device_connectors/devices/oemrecovery/oemrecovery.py

```diff
@@ -184,9 +184,19 @@ def copy_ssh_id(self):
     def check_device_booted(self):
         """Check to see if the device is booted and reachable with ssh"""
         logger.info("Checking to see if the device is available.")
+        try:
+            boot_timeout = self.job_data.get("test_data", {}).get(
```

Comment:
This is not something generally associated with the test phase, so the test_data section is probably the wrong place for it. We don't currently support an option for this, but if we did, it would probably go in either provision_data or up at the top level.